test/units/sh_yesno

```diff
@@ -27,7 +27,8 @@ for f in no NO No false FALSE False 0 ; do
 		echo "!$f!"
 	fi
 done
-: $(( ret += $tret ))
+eend $tret
```

Comment:
you didn't mean to add this line, so delete it

Reply:
Yep, again, good catch, thanks.